Source/DataSources/BillboardGraphics.js

```diff
@@ -46,7 +46,6 @@ define([
      * @param {Property} [options.imageSubRegion] A Property specifying a {@link BoundingRectangle} that defines a sub-region of the image to use for the billboard, rather than the entire image.
      * @param {Property} [options.sizeInMeters] A boolean Property specifying whether this billboard's size should be measured in meters.
      *
-     * @demo {@link http://cesiumjs.org/Cesium/Apps/Sandcastle/index.html?src=Billboards.html|Cesium Sandcastle Billboard Demo}
```

Comment:
This, and all of the demo links in the `DataSources` folder can stay.

We just wanted to remove the links from the core geometry types because the links are broken